types/redux-testkit/package.json

```diff
@@ -2,6 +2,6 @@
 	"private": true,
 	"dependencies": {
 		"redux": "^3.6.0",
-		"redux-thunk": "^2.2.0"
+		"redux-thunk": "2.2.0"
```

Comment:
Should be able to drop this line entirely, right? Then import the types from DT.